sdk/storage/azure-storage-file-share/azure/storage/fileshare/_directory_client.py

```diff
@@ -272,6 +272,33 @@ def create_directory(self, **kwargs):
         # type: (Any) -> Dict[str, Any]
         """Creates a new directory under the directory referenced by the client.
 
+        :keyword file_attributes:
```

Comment:
if these attrs are applied to files/dirs - I think the kwarg `file-attributes` could be changed to something else? I'm not sure why in other kwargs we prefix it with `file` :/

Reply:
Yeah, I agree the prefix isn't ideal but I think its best to leave it to be consistent with the rest of the APIs here. `set_http_headers()`, `rename_file()`, etc. already prefix all the SMB properties with file.

This is also consistent with the service documentation / header names since they are the same for files and directories.
https://docs.microsoft.com/en-us/rest/api/storageservices/create-directory#request-headers